doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -145,7 +145,7 @@ bash configure --with-boot-jdk=/home/jenkins/bootjdks/jdk8
 
 :pencil: Configuring and building is not specific to OpenJ9 but uses the OpenJDK build infrastructure with OpenJ9 added.
 Many other configuration options are available, including options to increase the verbosity of the build output to include command lines (`LOG=debug`).
-For more information see [OpenJDK build troubleshooting](https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk8u/master/README-builds.html#troubleshooting).
+For more information see [OpenJDK build troubleshooting](https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/doc/building.md#troubleshooting).
```

Comment:
I'm not sure it makes sense to switch from `.html` to `.md`; the old URL appears to be missing the `refs/heads/` segment and `README-builds.html` was relocated to `doc/building.html` so the URL should be:
https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk8u/refs/heads/master/doc/building.html#troubleshooting

Reply:
If you prefer, we can use URLs that refer to repositories in the ibmruntimes organization, e.g.:
https://htmlpreview.github.io/?https://github.com/ibmruntimes/openj9-openjdk-jdk8/refs/heads/openj9/doc/building.html#troubleshooting

Reply:
What's wrong with .md? The build instructions themselves are .md. It gets rid of using raw.githubusercontent.com and htmlpreview.github.io, and

Reply:
Elsewhere in the doc there is a link to https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md, which is also .md.

Reply:
It also appears the .md is the master and the html is auto generated from that.

Reply:
> It also appears the .md is the master and the html is auto generated from that.

Both files are from upstream, and seem to always be updated together, but I hadn't thought of that. What lead you to that conclusion?

Reply:
The last commit comment on jdk8 is [8176509: Use pandoc for converting build readme to html](https://github.com/ibmruntimes/openj9-openjdk-jdk8/commit/d33bd02ada68d13d8c886169390fdb062bb95aa3).

Reply:
There's no html alternative for OpenJ9TestUserGuide.md, else I might suggest that as well.

The `.md` approach clutters the view with stuff from github:

![image](https://github.com/user-attachments/assets/b16bb920-4bc4-486b-b876-1f1057af9d65)

If openjdk agreed with you, they wouldn't need to produce the html version.

Reply:
Updated.